src/web/components/ArticleHeadline.tsx

```diff
@@ -238,14 +231,7 @@ const renderHeadline = ({
             return (
                 // Inverted headlines have a wrapper div for positioning
                 // and a black background (only for the text)
-                <div
-                    className={cx(
-                        // We only shift the inverted headline down when main media is showcase
-                        isShowcase ? shiftPosition('down') : shiftSlightly,
-                        maxWidth,
-                        displayFlex,
-                    )}
-                >
```

Comment:
I moved this layout logic out of the headline component and put it in the layout files where it should have been to start with

Reply:
+1